main.go

```diff
@@ -88,7 +88,11 @@ func cloneRepo(cfg repoConfig) *step.Error {
 			SSHKeySavePath:          path.Join(pathutil.UserHomeDir(), ".ssh", "steplib_ssh_step_id_rsa"),
 			IsRemoveOtherIdentities: false,
 		}); err != nil {
-			return err
+			return newStepError(
```

Comment:
The `cloneRepo` function returns a `*step.Error`, which wraps a built-in error, so these two occurrences have to be modified to eliminate the type error.